wallet/src/main/resources/postgresql/wallet/migration/V17__create_wallet_state.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE wallet_state (id INT NOT NULL PRIMARY KEY DEFAULT(1), is_rescanning BOOL NOT NULL);
```

Comment:
Maybe we want to consider just adding this as a column to `state_descriptors` table?

https://github.com/bitcoin-s/bitcoin-s/blob/f71d3567ed21df6bb050ee184d6fe31b4369c654/wallet/src/main/scala/org/bitcoins/wallet/models/WalletStateDescriptorDAO.scala#L89